src/forgot-password/ForgotPasswordPage.jsx

```diff
@@ -1,98 +1,98 @@
-                    placeholder="username@domain.com"
-                    value={emailInput}
-                    onChange={e => handleOnChange(e)}
-                    style={{ width: '400px' }}
-                  />
-                </ValidationFormGroup>
-              </div>
-              <p className="mb-2">
-                {intl.formatMessage(messages['logisration.forgot.password.page.email.field.help.text'])}
-              </p>
-              <LoginHelpLinks page="forgot-password" />
+    <Formik
+      validationSchema={schema}
+      onSubmit={(values) => {
+        props.forgotPassword(values.email);
+      }}
+      initialValues={{
+        email: '',
+      }}
+    >
+      {({
+        handleSubmit,
+        handleChange,
+        values,
+        errors,
+      }) => (
+        <>
+          {status === 'complete' ? <Redirect to={LOGIN_PAGE} /> : null}
+          <div className="d-flex justify-content-center forgot-password-container">
+            <div className="d-flex flex-column" style={{ width: '450px' }}>
+              <Form className="m-4">
+                <div className="form-group">
+                  {status === 'forbidden' ? <RequestInProgressAlert /> : null}
+                  <h3 className="text-center mt-3">
+                    {intl.formatMessage(messages['logisration.forgot.password.page.heading'])}
+                  </h3>
+                  <p className="mb-4">
+                    {intl.formatMessage(messages['logisration.forgot.password.page.instructions'])}
+                  </p>
+                  <div className="d-flex flex-column align-items-start">
+                    <ValidationFormGroup
+                      className="mb-0"
+                      for="email"
+                      invalid={!!errors.email}
+                      invalidMessage={intl.formatMessage(
+                        messages['logisration.forgot.password.page.invalid.email.message'],
+                      )}
+                    >
+                      <label htmlFor="forgot-password-input" className="h6 mr-1">
+                        {intl.formatMessage(messages['logisration.forgot.password.page.email.field.label'])}
+                      </label>
+                      <Input
+                        name="email"
+                        id="forgot-password-input"
+                        type="email"
+                        placeholder="username@domain.com"
+                        value={values.email}
+                        onChange={handleChange}
+                        style={{ width: '400px' }}
+                      />
+                    </ValidationFormGroup>
+                  </div>
+                  <p className="mb-2">
+                    {intl.formatMessage(messages['logisration.forgot.password.page.email.field.help.text'])}
+                  </p>
+                  <LoginHelpLinks page="forgot-password" />
+                </div>
+                <StatefulButton
+                  type="submit"
```

Comment:
I believe with `onClick` handler, the type shouldn't be `submit` but instead `button`.